esbuild.dev.mjs

```diff
@@ -10,6 +10,7 @@ await esbuild.build({
   loader: {
     '.png': 'dataurl',
     '.svg': 'text',
+    '.mp3': 'base64',
```

Comment:
I guess this way we're bundling the audio as part of our javascript assets which is not required,

Reply:
Still has to be tested in prod